ricaun-io/ricaun.RevitTest
# Version 1.1.1

### Features
- `TestDiscoverer` enable log feature.
- `TestAdapter` only execute Application if trusted.
### Updated
- Update `ricaun.NUnit` to `1.3.1`
### Command
- Update TargetFramework added `net5`
- Update `JsonExtension` with `System.Text.Json`
### Console
- Update `ricaun.Revit.Installation` to `1.1.0` (Fix: #30)
### TestAdapter
- Update `TestDiscoverer` with `Log` feature
- Update `RevitTestConsole` with `RunTestReadWithLog` feature
- Add `ricaun.Security.WinTrust` package
- Add `TrustApplicationUtils` to check if application is trusted
- Add `DelayAfterExit` in `ProcessStart` with value `100`
### Fixed
- Fix Folder not exist (Fix: #30)
